Solutions/CohesitySecurity/Data Connectors/Helios2Sentinel/zip.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/zsh
+# This script is used to publish the 2 function apps into 2 zip files, for later azure deployment.
+# no input needed, and it could be run from any where.
+
+SCRIPT=$(realpath "$0")
+SCRIPTPATH=$(dirname "$SCRIPT")
+cd "$SCRIPTPATH"
+
+dotnet clean
+dotnet publish -c Release
+PUBLISH_DIR="./bin/Release/net6.0/publish/"
+
+find "$SCRIPTPATH"/ -type f -iname \*.csproj | while read ss; \
+do \
+    SCRIPT=$(realpath "$ss")
+    CSPROJDIR=$(dirname "$SCRIPT")
+    cd "$CSPROJDIR"
+
+    filename=$(basename -- "$ss")
+    extension="${filename##*.}"
+    filename="${filename%.*}"
+    cd "$PUBLISH_DIR"
+    zip -r ./"$filename".zip ./*
+    mv -fv ./"$filename".zip "$CSPROJDIR/../Package/"
+    cd ../
```

Comment:
I copied zip.sh and ran it my local directory. The following is the output

```
MacBook-Pro:Helios2Sentinel kishannerella$ pwd
/Users/kishannerella/software/Azure-Sentinel/Solutions/CohesitySecurity/Data Connectors/Helios2Sentinel
MacBook-Pro:Helios2Sentinel kishannerella$ git status
On branch internal
Your branch is up to date with 'origin/CohesitySecurity.internal'.

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	IncidentConsumer.zip
	zip.sh
	../../Package/IncidentProducer.zip
```
It looks like the consumer package is not at the right place.

Reply:
do you have realpath installed in your vm or macos?

Reply:
could you copy the output of the script running?

Reply:
realpath was not installed. After installing it, it's working fine.